godot-core/src/builtin/projection.rs

```diff
@@ -11,10 +11,12 @@ use sys::{ffi_methods, GodotFfi};
 use crate::builtin::inner::InnerProjection;
 use crate::builtin::math::{ApproxEq, GlamConv, GlamType};
 use crate::builtin::{real, Plane, RMat4, RealConv, Transform3D, Vector2, Vector4, Vector4Axis};
+use crate::engine::utilities::{atan, deg_to_rad, rad_to_deg, tan};
```

Comment:
Note that these are using Godot's implementation. 

We should probably use Rust's standard equivalents instead. There are even [functions to convert radians/degrees](https://doc.rust-lang.org/std/primitive.f64.html#method.to_degrees).